### PR TITLE
add spacing to project page

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

add visual consistentcy to projects page header
### before

After

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2365** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2365" target="_blank">Graphite</a> 
    * **PR #2403** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2403" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2403?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

**Comment by benschac** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2403?utm_source=stack-comment).


Job Reference: ezot5IR4ch6wNJ3FjohG

**Pull request merged**
